app/views/admin/shared/_sidebar.html.slim

```diff
@@ -14,4 +14,7 @@ aside
               span FAQs
           li 
             = link_to admin_payout_reports_path
-              span Payout Reports
\ No newline at end of file
+              span Payout Reports
+          li 
+            = link_to admin_unattached_promo_registrations_path
+              span Referral Promo
```

Comment:
Would you call it `Referral Promos`? It's an index of all of them

Reply:
Hm not sure `Referral Promos` is right here.  We're running one promotion, in which users and admins can generate multiple referral codes.

Reply:
Perhaps call it Referral Campaigns then?

Reply:
(I'm nit-picking lol. not too big of a deal)